js/runner.go

```diff
@@ -200,9 +200,13 @@ func (r *Runner) newVU(idLocal, idGlobal uint64, samplesOut chan<- stats.SampleC
 		MinVersion:         uint16(tlsVersions.Min),
 		MaxVersion:         uint16(tlsVersions.Max),
 		Certificates:       certs,
-		NameToCertificate:  nameToCert,
 		Renegotiation:      tls.RenegotiateFreelyAsClient,
 	}
+	// Follow NameToCertificate in https://pkg.go.dev/crypto/tls@go1.17.6#Config, leave this field nil
+	// when it is empty
+	if len(nameToCert) > 0 {
+		tlsConfig.NameToCertificate = nameToCert
```

Comment:
```suggestion
        r.Logger.Warn("the message here")
		tlsConfig.NameToCertificate = nameToCert
```
the runner has the logger so here could be an option

Reply:
I have tested, if we add a log here, because `newVU` will be called multiple times, the log will be printed multiple times in one run